path/glob_to_regexp.ts

```diff
@@ -9,9 +9,17 @@ import {
   globToRegExp as windowsGlobToRegExp,
 } from "./windows/glob_to_regexp.ts";
 
-export type { GlobOptions };
+export type { GlobOptions, OSType };
```

Comment:
Needed to resolve the lint that `OsType` is private despite appearing in a public type (`GlobToRegExpOptions`).

Alternatively we could inline the definition of `OSType` instead of exporting it, but that sort of defeats the purpose of the type.

Reply:
Exporting the type makes sense to me, but I think it should be renamed to `OsType`. The Deno style guide has the below section:

> When the names are in camelCase or PascalCase, always follow the rules of them even when the parts of them are acronyms.
>
> Note: Web APIs use uppercase acronyms (JSON, URL, URL.createObjectURL() etc.). Deno Standard Library does not follow this convention.

https://docs.deno.com/runtime/manual/references/contributing/style_guide#naming-convention